### ISSUE TITLE
Replacing images for texturing with a single image?

### ISSUE THREAD

**Issue by RocketGoose** [alicevision/meshroom#1711]
I'm trying to follow a similar workflow as described here [http://dlib.nyu.edu/awdl/isaw/isaw-papers/21/](url) (but using meshroom instead of metashape) where the base images for texturing are swapped with an RTI normal map. 

I've tried this method [https://github.com/alicevision/meshroom/wiki/Projected-Light-Patterns](url) for swapping textures, but it requires the same number of images in the replacement image folder as the base images folder. I only have one image (the normal map) in the replacement folder so I can't figure out how to make this work. 

Is there a method for this? I also tried tried filling the rest of the replacement folder with plain black images so there would be the correct number of them, but I got this error. 
![image](https://user-images.githubusercontent.com/40300361/173047981-2a9e4180-2e76-46b3-94b4-ceb198b30423.png)

**Comment by fabiencastan** [comment]
Yes, currently it requieres to have an image for each camera. But that's a good point, it could make sense to add an option for that.

In the meantime, you shoud be able to create an sfmData with only the pose that you want and use that to do the texturing. That's not user friendly to do it at all, but you have 2 options:
- Option 1: export to .sfm (using ConvertSfMData) and modify this json text file manually
- Option 2: in the UI
  - create a new CameraInit (in the ImageGallery you can switch from one CameraInit to another)
  - drag&drop the selected original source image into it
  - Add an SfMTransfer to retrieve the pose for the selected view
  - Use this sfmData (the output of the SfMTransfer) as the input for the Texturing

**Comment by RocketGoose** [comment]
Hi, I'm returning to this project again and any help would be appreciated! [fabiencastan](https://github.com/fabiencastan), thanks for the suggestion, It seems to be working but I'm running into a problem with texture atlases. 
The second texturing node is generating a single texture atlas while the original model has 4. As you can see bellow, the new texture doesn't line up with the UVs. I've sent my current node structure in case I'm doing something obviously wrong 
![image](https://user-images.githubusercontent.com/40300361/234543242-471baa2b-e11f-4630-9403-69f9ba5939d2.png)
![image](https://user-images.githubusercontent.com/40300361/234543460-9c705cd5-74cb-4e23-8987-3d5eb98943d0.png)

**Comment by RocketGoose** [comment]
So as it turned out my node setup was wrong. I'm leaving a screenshot of the corrected version in case it helps anyone.

Some issues with this approach:
1. The resulting mesh will be missing any geometry not visible in the cameras of the new camera init
2. When there is lots of missing geometry the final texturing step crashes after saving the new mesh, but before creating the texture. I've actually been projecting several normal map from different directions to cover most of the model and it seems to work when most of the model is covered and there is not much missing geometry. However, if I slowly reduce coverage (number of images in the new camera init) there comes a point where the texturing step crashes.


![image](https://github.com/alicevision/Meshroom/assets/40300361/2a9f62c8-b8d1-44ee-91ba-a767761eb445)
